src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -11,36 +11,33 @@
 
 namespace Symfony\Component\Form\Extension\Core\Type;
 
-use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\ChoiceList\Loader\CallbackChoiceLoader;
 use Symfony\Component\Intl\Intl;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class CountryType extends AbstractType
+class CountryType extends ImmutableChoiceType
 {
     /**
      * {@inheritdoc}
      */
-    public function configureOptions(OptionsResolver $resolver)
-    {
-        $resolver->setDefaults(array(
-            'choices' => array_flip(Intl::getRegionBundle()->getCountryNames()),
-            'choice_translation_domain' => false,
-        ));
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getParent()
+    public function getBlockPrefix()
     {
-        return __NAMESPACE__.'\ChoiceType';
+        return 'country';
```

Comment:
I mean you might be looking at the diff "unified", and should try "split", because this method is not changed at all :)